pages/login.page.js

```diff
@@ -0,0 +1,38 @@
+import BasePage from "./base.page";
+const PAGE_TIMEOUT = 10000;
+const BASE_PAGE = BasePage;
+
+class LoginPage {
+    
+    get usernameInput () { return $('#username'); }
+    get passwordInput () { return $('#password'); }
+    get rememberMe () { return $('#remember-me'); }
+    get logInButton () { return $('#submit'); }
+    get failedLoginMessage () {return $('[class= "badge badge-pill badge-danger"]'); }
+
+    async navigateToPage(){
+        await BASE_PAGE.open('login');
+    }
+    
+    async completeCredentials(username, password) {
+        await BASE_PAGE.completeFiels(await this.usernameInput, username);
```

Comment:
Same comment as above, after inheritance is implemented, you use super instead of BASE_PAGE to access this method